core/src/executor/evaluate/evaluated.rs

```diff
@@ -220,6 +220,10 @@ impl<'a> Evaluated<'a> {
         binary_op(self, other, |l, r| l.modulo(r), |l, r| l.modulo(r))
     }
 
+    pub fn shift_left<'b>(&'a self, other: &Evaluated<'b>) -> Result<Evaluated<'b>> {
```

Comment:
as we'd better to use more explicit name for binary operator `BitwiseShiftLeft`, this method name would be better to use `bitwise_shift_left`.